jenkins/databricks/test.sh

```diff
@@ -33,10 +33,25 @@ sudo chmod 777 /databricks/data/logs/
 sudo chmod 777 /databricks/data/logs/*
 echo { \"port\":\"15002\" } > ~/.databricks-connect
 
+SPARK_SUBMIT_FLAGS="--conf spark.python.daemon.module=rapids.daemon_databricks \
+    --conf spark.rapids.memory.gpu.allocFraction=0.1 \
```

Comment:
I assume this setting still works for the rest of the tests, did we see a change in runtime at all?

Reply:
oh never mind, I see below we run them separate

Reply:
it might be nice to rename these to be CUDF_UDF_TEST_ARGS

Reply:
Good suggestion, let me change it